Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM library/node:6.3
+
+WORKDIR /src
+
+COPY package.json .
+
+RUN npm install
+
+COPY . .
+
+RUN node_modules/bower/bin/bower --allow-root install \
```

Comment:
nope, scripts configured in package.json by default have `node_modules/.bin` prepended to PATH when executed with `npm run `